app/services/citizen_email_service.rb

```diff
@@ -32,4 +32,8 @@ def applicant
     # application.
     @applicant ||= application&.applicant
   end
+
+  def secure_id
+    application.generate_secure_id
```

Comment:
❓ Isn't this code and [the email form controller](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/14cc44122efdd89d144bc3db5a8e52e72dbf0946/app/controllers/providers/applicants_controller.rb#L65) generating two distinct secure IDs? Is this intended or once the secure ID is generated once it should stand as the only one (unless we forcefully re-generated it)?

Reply:
That's correct. In the final code it should only be needed once. I've added a comment to the applicants controller code stating that this should be removed when we no longer need an easy way to display a working link.